hawkular-alerts-api/src/main/java/org/hawkular/alerts/api/services/DefinitionsService.java

```diff
@@ -74,9 +175,13 @@
         i.e. querying for properties to fill for a specific notifier type.
      */
     void addNotifierType(String notifierType, Set<String> properties) throws Exception;
```

Comment:
I am +1 to change the Notifier / Notifications terminology.
Without enter in a full discussion, I would start by the name.
I like the concept of Action instead of Notifier as you mentioned in some discussion, and I would propose ActionPlugin as NotifierType.
We can start with these changes (and propagating into the whole hawkular-alerts) maintaining current structure, and think/change structure later.
What do you think ?